specs/_features/eip7732/fork-choice.md

```diff
@@ -39,31 +38,32 @@
 
 This is the modification of the fork choice accompanying the EIP-7732 upgrade.
 
+## Custom types
+
+| Name            | SSZ equivalent | Description                                     |
+| --------------- | -------------- | ----------------------------------------------- |
+| `PayloadStatus` | `uint8`        | possible status of a payload in the fork-choice |
+
 ## Constants
 
-| Name                           | Value                   |
-| ------------------------------ | ----------------------- |
-| `PAYLOAD_TIMELY_THRESHOLD`     | `PTC_SIZE // 2` (= 256) |
-| `INTERVALS_PER_SLOT`           | `4`                     |
-| `PROPOSER_SCORE_BOOST_EIP7732` | `20`                    |
-| `PAYLOAD_WITHHOLD_BOOST`       | `40`                    |
-| `PAYLOAD_REVEAL_BOOST`         | `40`                    |
+| Name                       | Value                   |
+| -------------------------- | ----------------------- |
+| `PAYLOAD_TIMELY_THRESHOLD` | `PTC_SIZE // 2` (= 256) |
+| `INTERVALS_PER_SLOT`       | `4`                     |
+| `PAYLOAD_STATUS_PENDING`   | `PayloadStatus(0)`      |
```

Comment:
If we're using this construct on the node, it may be good also to have the same structure on the PTC attestation, so that PTC members can vote with `PAYLOAD_STATUS_PENDING` in case the beacon block is missing. 

Can't really see a good use for this right now so just annotating this in this review for lack of better place.